### DISCUSSION TITLE
Nucleo-L073rz How to blink the onboard led

### DISCUSSION THREAD

**Discussion by speedyant71** [micropython/micropython#10265] [STM32 / Pyboard] [answered]
How to blink the onboard led marked LD2?

**Comment by jimmo** [comment, marked as answer]
@speedyant71 It's available via pyb.LED(1) like any other STM32 / pyboard board.
You can also access the pin directly via machine.Pin.board.D13 or machine.Pin.cpu.A5.

**Reply by speedyant71** [reply]
Thanks a lot!
